File: year2020/month08/day0814/ms_0814.cpp
#include <iostream>

using namespace std;

int main() {
	int data[100], i = 0;

	while (i < 100) {
		int temp;
		
		scanf_s("%d", &temp);

		if (temp != 0) {
			data[i] = temp;
			i++;
		}
		else break;
	}

	int count = 0, sum = 0;
	
	for (int j = 0; j < i; j++) {
		if (data[j] % 5 == 0) {
			sum += data[j];
			count++;
		}
	}

	cout << "Multiples of 5 : " << count << endl;
	cout << "sum : " << sum << endl;
	printf("avg : %.1lf", (sum / double(count)));

	return 0;
}

File: year2020/month08/day0814/ms_0814.cs
using System;

namespace algorithm_35
{
    class ms_0814
    {
        static void Main(string[] args)
        {
            int[] data = new int[100];
            int count = 0, i = 0;

            while(i < data.Length) 
            {
                int temp = Convert.ToInt32(Console.ReadLine());

                if(temp != 0) 
                {
                    data[i] = temp;
                    count++;
                }
                else break;

                i++;
            }

            int sum = 0, multiples = 0;

            for(i = 0; i < count; i++) 
            {
                if(data[i]%5 == 0) 
                {
                    sum += data[i];
                    multiples++;
                }
            }

            double avg = sum / Convert.ToDouble(multiples);

            Console.WriteLine($"Multiples of 5 : {multiples}");
            Console.WriteLine($"sum : {sum}\navg : {avg:.0}");
        }
    }
}
